monkey/common/utils/exceptions.py

```diff
@@ -56,3 +56,7 @@ class DomainControllerNameFetchError(FailedExploitationError):
 
 class InvalidConfigurationError(Exception):
     """ Raise when configuration is invalid """
+
+
+class IslandCommunicationError(Exception):
+    """Raise when unable to connect to control client"""
```

Comment:
This doesn't need to be in `common`. It can just be defined in `IControlChannel`